package/MDAnalysis/topology/PDBParser.py

```diff
@@ -303,6 +308,22 @@ def _parseatoms(self):
         masses = guess_masses(atomtypes)
         attrs.append(Masses(masses, guessed=True))
 
+        # Guessed attributes
+        # Need to pull elements from Atom names
+        # Similar to the check for atomtypes function
+        if not any(elements):
+            # Can further add a check_cg_atom(elements) function in guessers.py to check for CG atom.
+            elements = guess_atom_element(elements)
+            if elements == '':
+                warnings.warn("Element record found to be non-physical.")
+                #Nomenclature that X will denote any non-physical element.
+                elements = 'X'
+            attrs.append(Elements(elements, guessed=True))
+        else:
+            attrs.append(Elements(guessers.guess_types(elements),
+                                  guessed=True))
```

Comment:
Why are you guessing the elements again?

Reply:
I did it as a double check. I was think of providing a None/False in the else loop